src/pages/guides/how-tos/enhancing-audio-files-with-enhance-speech-api.md

```diff
@@ -0,0 +1,275 @@
+
+[esqa-link]:https://podcast.adobe.com/enhance
+
+## Enhance Speech API Walkthrough
+
+### Step 0: Obtain a valid access token
+
+Ensure you have reference to a valid access token. See the [Authentication Guide](../concepts/authentication/index.md) for details on how to obtain
+such a token.
+
+### Step 1: Create an asset
+
+Before we can run speech enhancement on an `Asset`, we need to register a new `Asset` with the Enhance Speech API.
+To accomplish this, we use the `POST /audio_services/v1/assets` endpoint.
+
+This endpoint requires, as a JSON request body, the following information:
+
+* `filename`:
+    * Description: A filename (with extension) of the asset. We use this filename when naming the downloaded file for a downstream `SpeechEnhancement`.
+    * Example: "earhart-aviation.mp3"
+* `byte_size`:
+    * Description: The number of bytes of the asset.
+    * Example: 59400
+* `checksum`
+    * Description: The base64-encoded MD5 checksum of the asset.
+    * Example: "AWzJTRSXGy5JsL+Phh/cFA=="
+* `content_type`
+    * Description: The content type of the asset.
+    * Example: "audio/mpeg"
+
+We use the `byte_size`, `checksum` and `content_type` to enable secure and verifiable direct uploads of your asset to our object store.
+
+Putting this all together. Using the curl HTTP client, we can create an asset:
+
+```sh
+curl 'https://firefly-api-enterprise-stage.adobe.io/audio_services/v1/assets' \
+--header 'x-api-key: {CLIENT_ID}' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer {ACCESS_TOKEN}' \
+--data '{
+    "filename": "earhart-aviation.mp3",
+    "byte_size": 59400,
+    "checksum": "AWzJTRSXGy5JsL+Phh/cFA==",
+    "content_type": "audio/mpeg"
+
+}'
+```
+
+If all goes well, we'll get a `201 Created` response, with a payload like the following:
+
+```json
+{
+    "id": "3445d7a0-2513-4606-b37b-bff0034c0214",
+    "filename": "earhart-aviation.mp3",
+    "upload_url": "https://phonos-recordings-staging.s3-accelerate.amazonaws.com/Assets/3445d7a0-2513-4606-b37b-bff0034c0214?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAS2MU72M7ACPXX764%2F20240409%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240409T213836Z&X-Amz-Expires=43200&X-Amz-Security-Token=FwoGZXIvYXdzEPf%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaDMyjVxJyIRosE7smGSLJAWqfi8jqgkF0iiCYFU1nq3xAzNalFsrmbNnJRpWZvOSjGREvQftcLr%2FVgG9rjuz1cskXD3H8umxVLOS6oVmjHdt0wmaRtx5C6ZvGp8tW7BL%2F0hsuuU%2FmmG0n8QmmuQjA%2BfeaRkwR6Fgw85SryaPZUxzXhX9Y2BypyeBDX1H5JTPUYpZA0fAu1g9nYncsIjbNgnBJvr5e42gBnJZQSyAeeE3dA%2BK9btul5QapF5of4JTHXngkhLSBm819JD9iXRjRiUAUfoofGYfDhiiw5NawBjItFfJCqsABzKqETJ6q87zlAHajnVODPJgGowODSXrrqNd%2BTnV%2B73UXZTYKNukt&X-Amz-SignedHeaders=content-length%3Bcontent-md5%3Bcontent-type%3Bhost&X-Amz-Signature=45bad70aa97e89bf6120a7a0ec22b0dfc4771060922c50e1ec5c876f13252c49",
```

Comment:
FYI – this **was** short lived, staging, direct upload URLs that are no longer valid